services/utils/tests/unit_tests/utils_test.py

```diff
@@ -3,7 +3,8 @@
 import contextlib
 import json
 from aiohttp.test_utils import make_mocked_request
-from services.utils import format_qs, format_baseurl, DBConfiguration, handle_exceptions
+from services.utils import format_qs, format_baseurl, DBConfiguration
+from services.metadata_service.api.utils import handle_exceptions
```

Comment:
speaking of circular dependencies, now that the `handle_exceptions` was moved to the metadata services api/utils, consider moving the corresponding tests to `services/metadata_service/tests/unit_tests/` instead.